go.sum

```diff
@@ -1395,6 +1397,8 @@ github.com/tektoncd/pipeline v0.11.0/go.mod h1:hlkH32S92+/UODROH0dmxzyuMxfRFp/Nc
 github.com/tektoncd/pipeline v0.13.1-0.20200625065359-44f22a067b75/go.mod h1:R5AlT46x/F8n/pFJFjZ1U1q71GWtVXgG7RZkkoRL554=
 github.com/tektoncd/pipeline v0.17.1-0.20201006183654-d5df1c164a48 h1:bREVQct2dVm96b5nD2kJ6THuIoko7wv0jfp4xX9yxlo=
 github.com/tektoncd/pipeline v0.17.1-0.20201006183654-d5df1c164a48/go.mod h1:KPpPkbruz7vAHu5Y8dzu73IkgmR9Ew1HNkI3IZHbJ8k=
```

Comment:
Maybe we need to remove tekton `0.17.1` package.

Reply:
@xiujuan95 awesome, yes indeed. Fixed